### PR TITLE
Improve readmes

### PR DESCRIPTION
**Pull request by emmenko**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,6 @@
     "design-tokens:build": "yarn --cwd design-system build:tokens",
     "design-tokens:build:watch": "yarn --cwd design-system build:tokens:watch",
     "prebuild": "yarn generate-icons && yarn design-tokens:build && yarn i18n:compile && yarn generate-readmes",
-    "prestorybook:start": "yarn prebuild",
```
Comment (emmenko) [inline package.json:24]: As suggested by @jonnybel.
Didn't want to open a separate PR for this.

File: .eslintrc.yaml [hunk 2]
```diff
@@ -34,6 +34,7 @@ overrides:
   - files: ['*.spec.js', '**/docs/*.js']
     rules:
       'react/display-name': 0
+      'react/prop-types': 0
```
Comment (adnasa) [inline .eslintrc.yaml:37]: why?

**Review by adnasa — APPROVED** [review]

File: .eslintrc.yaml (hunk 2 above)
Comment (emmenko) [reply .eslintrc.yaml:37]: In test files and usage example files I find it's not really necessary to have, it's just more boilerplate.

Comment (tdeekens) [reply .eslintrc.yaml:37]: I agree. It's about the test cases less about always being complete. Plus testing for incomplete `prop-types` has a propose.

File: package.json (hunk 1 above)
Comment (jonnybel) [reply package.json:24]: Thanks!

**Review by jonnybel — APPROVED** [review]

**Pull request merged**